packages/dx-react-grid/docs/reference/table-filter-row.md

```diff
@@ -94,10 +94,10 @@ Describes properties passed to a component that renders a toggle button for a fi
 
 Field | Type | Description
 ------|------|------------
-value | any | The current editor value.
 disabled | boolean | Specifies whether the editor is disabled.
```

Comment:
The `disabled` field is not required too. Please add `?` after a name of the property.